File: exo6.php
<?php
$program_loop = false;
$choice_loop = false;
$words = "";
$userInput = "";
while ($program_loop == false){
  $words = readline("What should the parrot say ? ");
  echo $words . "\n";
  $choice_loop = true;
  while ($choice_loop == true){
    $userInput = readline("Enter 'R' to restart or 'X' to exit. ");
    if ($userInput == "X" || $userInput == "x"){
        echo "Goodbye.\n";
        $choice_loop = false;
        $program_loop = true;
    }elseif ($userInput == "R" || $userInput == "r"){
        $choice_loop = false;
    }else{
        echo "Wrong input.\n";
    }
  }
}
?>

File: exo6.py
program_loop = False
choice_loop = False
words = ""
userInput = ""

while program_loop == False :
    words = raw_input("What should the parrot say ? ")
    print words
    choice_loop = True
    while choice_loop == True :
        userInput = raw_input("Enter 'R' to restart or 'X' to exit. ").capitalize()
        if userInput == "X":
            print('Goodbye.')
            choice_loop = False
            program_loop = True
        elif userInput == "R":
            choice_loop = False
        else :
            print('Wrong input.')
